src/app/shared/prevent-page-leave-modal/prevent-leave-page-modal.component.spec.ts

```diff
@@ -40,10 +37,10 @@ describe('AdrReasonModalComponent', () => {
     });
   });
 
-  describe('save', () => {
+  describe('on action', () => {
     it('should close the modal when called', () => {
       spyOn(component.dialogRef, 'close');
```

Comment:
can we add an assertion that checks that it is false before we click on action which then toggles the value?
Also some more description in the unit test will be good for understanding it, maybe something like:
"it toggles the dialogRef.close value to true when action() is clicked"

Reply:
I think it would be a good idea to decide first upon the reusable modal components and extract them to the common branch before making any more amends on the existing ones.

Reply:
As agreed, technical task raised as this was done prior to ADR and require/depends on a few changes of 10163.

Ref: https://jira.dvsacloud.uk/browse/CVSB-15099